portfolio/src/main/java/com/google/sps/data/CommentList.java

```diff
@@ -0,0 +1,40 @@
+
+package com.google.sps.data;
+import java.util.Date;
+import java.util.ArrayList;
+import java.util.List;
+
+//list of comments sorted by date
+public class CommentList{
+
+    private ArrayList<Comment> comments;
+    private int numComments; //used for gson purposes
```

Comment:
numComments could be easily inferred from comments.size() right? Do we need a separate variable for that?